kebechet/managers/info/info.py

```diff
@@ -28,13 +28,18 @@
 _INFO_ISSUE_NAME = 'Kebechet info'
 
 _LOGGER = logging.getLogger(__name__)
+# Github and Gitlab events on which the manager acts upon.
+_EVENTS_SUPPORTED = ['issues', 'issue']
 
 
 class InfoManager(ManagerBase):
     """Manager for submitting information about running Kebechet instance."""
 
     def run(self) -> typing.Optional[dict]:
         """Check for info issue and close it with a report."""
+        if self.parsed_payload.get('event') not in _EVENTS_SUPPORTED:
```

Comment:
From design/semantics PoV, it feels like the payload is parameter of a manager instead of its attribute. WDYT?

It might be easier to manipulate with managers later on (e.g. create different triggers of a manager).

Reply:
Are we supporting the cron-job run also? or the webhook is the only behaviour we need to support? 
I think we can create shallow copies of the payload from the manager base class, and work on it. I think this way the function defination's stay the same to original design, like just run(self) is all the managers need to implement. But we can change to pass the payload..

Reply:
@fridex I think I need to add the condition - 
`if parsed_payload:` for now to keep supporting all the managers on cron job.